src/amo/pages/Addon/index.js

```diff
@@ -398,6 +398,24 @@ export class AddonBase extends React.Component {
     );
   }
 
+  getMetaDescription() {
+    const { addon, i18n } = this.props;
+
+    if (!addon) {
+      return null;
+    }
+
+    const content = i18n.sprintf(
+      i18n.gettext('Download %(addonName)s for Firefox. %(summary)s'),
+      {
+        addonName: addon.name,
+        summary: addon.summary,
```

Comment:
do we still want to factor in persona which has descriptions for now? see https://github.com/mozilla/addons-frontend/pull/6069#discussion_r214071501

Reply:
I think it is fine as is, because we are about to stop supporting lightweight themes.